src/auth/choicewindow.cpp

```diff
@@ -0,0 +1,89 @@
+#include "choicewindow.h"
+#include "downloaderWrapper.h"
+
+#include <QLabel>
+#include <QListWidget>
+#include <QVBoxLayout>
+#include <QDialogButtonBox>
+#include <QAbstractButton>
+#include <QPushButton>
+#include <QDebug>
+#include <QApplication>
+#include <QDesktopWidget>
+
+ChoiceWindow::ChoiceWindow(QStringList &usersFilesList, QWidget *pParent)
+    : QDialog (pParent)
+{
+    window()->setFixedSize( window()->sizeHint() );
+    setModal(true);
+    setWindowTitle("Sign in, please");
```

Comment:
I suppose, it's better to create cost strings for string literal like these

Reply:
done